.travis.yml

```
@@ -36,3 +36,7 @@ matrix:
         - set -x
         - yarn deployment-build
         - yarn end-to-end-test
+    - name: lockfile-check
+      cache: yarn
+      script:
+        - ./scripts/yarn-lockfile-check.sh
```

```suggestion
        - ./scripts/ci-yarn-lockfile-check.sh
```
because of consistency and because this is the kind of crappy operations humans shouldn't need to worry about locally.